pointers/pointers.c

```diff
@@ -28,7 +32,15 @@ void string_copy(char *x, char *y)
 */
 char *find_char(char *str, int c)
 {
-
+    while (*str != '\0') {
+        if (*str == 'c') {
```

Comment:
c is an integer type.  What we want is the character that c represents on the ASCII table.  You can do that by simply referencing the c when comparing it with a char.  This tells C compiler that we are trying to match a char with char, hence it will treat c as a char instead of an int.  If you put single quotations around c, you are actually comparing *str to the letter 'c'.